resources/nfdi4bioimage.yml

```
@@ -14512,4 +14512,532 @@ resources:
+  - Anne L. Martel
+  description: 'Breast cancer (BC) is the second most commonly diagnosed cancer in the U.S. with more than 250,000 new cases of invasive breast cancers reported in 2017. The majority of women with locally advanced and a subset of patients with operable breast cancer will undergo systemic therapy prior to their surgery (neoadjuvant therapy/ NAT) to reduce the size of tumor(s) and possibly further undergo breast conserving surgery. The Post-NAT-BRCA dataset is a collection of representative sections from breast resections in patients with residual invasive BC following NAT. Histologic sections were prepared and digitized to produce high resolution, microscopic images of treated BC tumors. Also included, are clinical features and expert pathology annotations of tumor cellularity and cell types. The Residual Cancer Burden Index (RCBi), is a clinically validated tool for assessment of response to NAT associated with prognosis. Tumor cellularity is one of the parameters used for calculating the RCBi. In this dataset, tumor cellularity refers to a measure of residual disease after NAT, in the form of proportion of malignant tumor inside the tumor bed region; also annotated. (See MD Anderson RCB Calculator for a detailed description of tumor cellularity.) Malignant, healthy, lymphocyte and other labels were also provided for individual cells to aid development of cell segmentation algorithms.'
+  license: CC-BY-3.0
+  name: Assessment of Residual Breast Cancer Cellularity after Neoadjuvant Chemotherapy using Digital Pathology
+  publication_date: '2017-10-04'
+  tags:
+  - AI-ready
+  type:
+  - Data
+  url: https://www.cancerimagingarchive.net/collection/post-nat-brca/
+
+- authors:
+  - Mohamed Amgad 
+  - Habiba Elfandy 
+  - Hagar Hussein 
+  - Lamees A Atteya 
+  - Mai A T Elsebaie 
+  - Lamia S Abo Elnasr 
+  - Rokia A Sakr 
+  - Hazem S E Salem 
+  - Ahmed F Ismail 
+  - Anas M Saad 
+  - Joumana Ahmed 
+  - Maha A T Elsebaie 
+  - Mustafijur Rahman 
+  - Inas A Ruhban 
+  - Nada M Elgazar 
+  - Yahya Alagha 
+  - Mohamed H Osman 
+  - Ahmed M Alhusseiny 
+  - Mariam M Khalaf 
+  - Abo-Alela F Younes 
+  - Ali Abdulkarim 
+  - Duaa M Younes 
+  - Ahmed M Gadallah 
+  - Ahmad M Elkashash 
+  - Salma Y Fala 
+  - Basma M Zaki 
+  - Jonathan Beezley 
+  - Deepak R Chittajallu 
+  - David Manthey 
+  - David A Gutman
+  - Lee A D Cooper
+  description: 'This repo contains the necessary information and download instructions to download the dataset associated with the paper: Amgad M, Elfandy H, ..., Gutman DA, Cooper LAD. Structured crowdsourcing enables convolutional segmentation of histology images. Bioinformatics. 2019. doi: 10.1093/bioinformatics/btz083. This data can be visualized in a public instance of the Digital Slide Archive at this link. If you click the “eye” image icon in the Annotations panel on the right side of the screen, you will see the results of a collaborative annotation.'
+  license: MIT License
```

```suggestion
  license: CC0
```

According to this section, the data is CC0 licensed: https://github.com/PathologyDataScience/BCSS?tab=readme-ov-file#dataset-licensing

---

Yes, you are right. I just saw the licensing of the repo itself. Thanks for letting me know!